package.json

```diff
@@ -4,7 +4,8 @@
   "description": "Tupelo JavaScript client",
   "main": "lib/tupelo.js",
   "scripts": {
-    "test": "scripts/integration-tests"
+    "test": "scripts/integration-tests",
+    "docs": "esdoc"
```

Comment:
Might be safer to do `./node_modules/.bin/esdoc` since everyone might not have it global or $PATH mod?

Reply:
@brandonwestcott NPM automatically [resolves](https://docs.npmjs.com/misc/scripts#path) the path to binaries of dependencies (in this case esdoc), so this works the same as `./node_modules/.bin/esdoc`. I actually don't have esdoc installed globally myself, and it works.